### PR TITLE
docs: add custom dir placeholder explainer to installation guide

### PR DESCRIPTION
**Pull request by malept**
#### Description of Change

Now that electron/get#143 has been released, the user docs for installing Electron should explain how it works for mirrors.

CC: @electron/wg-ecosystem 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `yarn lint:docs` passes
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: docs/tutorial/installation.md [hunk 1]
```diff
@@ -69,6 +69,11 @@ For instance, to use the China mirror:
 ELECTRON_MIRROR="https://cdn.npm.taobao.org/dist/electron/"
 ```
 
+By default, `ELECTRON_CUSTOM_DIR` is set to `v$VERSION`. To change the format,
+use the `{{ version }}` placeholder. For example, `version-{{ version }}`
```
Comment (erickzhao) [inline docs/tutorial/installation.md]: > To change the format, use the `{{ version }}` placeholder

Nit: As someone who hasn't used `@electron/get` before, this phrasing is a bit vague to me. Does it mean that to set the dir to `version-5.0.0`, you should:

1) `ELECTRON_CUSTOM_DIR="version-{{ version }}"`
2) `url = ELECTRON_MIRROR + version-{{ version }} + '/' + ELECTRON_CUSTOM_FILENAME`
3) Either?

I'm leaning more towards #1 from what I'm understanding, but I think making it explicit would help.

Comment (malept) [reply docs/tutorial/installation.md]: It's 1. Option 2 is kind of implied. I'll see if I can make it clearer.

Comment (malept) [reply docs/tutorial/installation.md]: My attempt at clarity is in https://github.com/electron/electron/pull/22491/commits/520dcd2780412df91ce8c4ce2794ec845f6a3916

Comment (erickzhao) [reply docs/tutorial/installation.md]: :shipit:

**Pull request merged**